app/src/docs/components/DocsPages/Products/DataUnions/dataUnions.test.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import DataUnions from '.'
+
+jest.mock('$docs/components/DocsHelmet', () => ({
```

Comment:
This tests that the "Data unions coming soon" text won't be replaced by the new text.